packages/nameguard-react/src/components/ReportBadge/index.tsx

```diff
@@ -35,7 +35,7 @@ interface ReportBadgeProps {
     than this number, the badge will truncate the ensName and display a 
     tooltip on hover that shows the full ensName. This number is measured in pixels.
   */
-  maxEnsNameDisplayWidth?: number;
+  maxDisplayWidth?: number;
 }
 
 const DEFAULT_MAX_ENSNAME_DISPLAY_WIDTH = 200;
```

Comment:
This constant should be removed. This idea belongs inside `DisplayedName` not `ReportBadge`. Please let me know if you think I'm missing something :+1: